docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+
+services:
+  db:
+    container_name: pg_container
+    image: postgres
+    restart: always
+    environment:
```

Comment:
Should these db's env variables be public here?

Reply:
This should not be problem